<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsSelect</class>
 <widget class="QDialog" name="ControlsSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #24261f;</string>
  </property>
  <widget class="QLabel" name="lbl_player">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>40</y>
     <width>136</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0b93a5;</string>
   </property>
   <property name="text">
    <string>Player x</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0b93a5;
background-color: #3a3d34;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Use mouse</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="img_right">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="img_left">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="img_action">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_left">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>140</y>
     <width>93</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0b93a5;</string>
   </property>
   <property name="text">
    <string>Turn left</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_right">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>140</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0b93a5;</string>
   </property>
   <property name="text">
    <string>Turn right</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_action">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>140</y>
     <width>67</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0b93a5;</string>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_left">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>260</y>
     <width>64</width>
     <height>110</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:  72px;
background: transparent;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_right">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>260</y>
     <width>69</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 72px;
background: transparent;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_action">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>260</y>
     <width>52</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 72px;
background: transparent;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="err_assigned">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>276</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
background: transparent;
visibility: hidden;</string>
   </property>
   <property name="text">
    <string>Button already assigned !</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mute">
   <property name="geometry">
    <rect>
     <x>638</x>
     <y>494</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
